guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleConnection.java

```diff
@@ -157,6 +162,43 @@ public SimpleConnection(String name, String identifier,
         this.fullConfig = config;
         this.interpretTokens = interpretTokens;
 
+    }
+    
+        /**
+     * Creates a new SimpleConnection having the given identifier,
+     * GuacamoleConfiguration, and GuacamoleProxyConfiguration. Parameter tokens
+     * will be interpreted if explicitly requested.
+     *
+     * @param name
+     *     The name to associate with this connection.
+     *
+     * @param identifier
+     *     The identifier to associate with this connection.
+     *
+     * @param config
+     *     The configuration describing how to connect to this connection.
+     *
+     * @param interpretTokens
+     *     Whether parameter tokens in the underlying GuacamoleConfiguration
+     *     should be automatically applied upon connecting. If false, parameter
+     *     tokens will not be interpreted at all.
+     * 
+     * @param proxyConfig
+     *     The Guacamole proxy configuration used to tell the client how to
+     *     connect to guacd.
```

Comment:
As it may be useful to invoke this constructor for cases where the default environment is desired (rather than conditionally invoke the other constructor to ensure defined behavior), I think we should add an "..., or null to ..." bit to explicitly document that behavior.

Reply:
Sure, sounds good.